File: sample-code/sample-bot.js
const Discordie = require('discordie');
const Events = Discordie.Events;
const discordClient = new Discordie();
const BOT_TOKEN = 'your_token_here';

discordClient.connect({
  token: BOT_TOKEN,
});

discordClient.Dispatcher.on(Events.GATEWAY_READY, e => {
  console.log(discordClient.User.username + ' connected!');
});

discordClient.Dispatcher.on(Events.MESSAGE_CREATE, ({message}) => {
  messag.channel.sendMessage('You are awesome!');
});

File: sample-code/sample-bot.cs
using Discord;
class Program
{
    static void Main(string[] args) => new Program().Start();

    public void Start()
    {
        var client = new DiscordClient();

        client.MessageReceived += async (s, e) =>
        {
            if (!e.Message.IsAuthor)
                await e.Channel.SendMessage($"{e.Message.User.Mention} is awesome!");
        };

        client.ExecuteAndWait(async () => {
            await client.Connect("MY_BOT_TOKEN_HERE", TokenType.Bot);
        });
    }
}